src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifierFactory.java

```diff
@@ -50,18 +50,18 @@
  *
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
-public class NotificationFactory {
-    private static final Logger logger = LoggerFactory.getLogger(NotificationFactory.class);
-    private static NotificationFactory instance;
+public class GerritNotifierFactory {
```

Comment:
Renaming a public class as this is generally frowned upon in Jenkins plugins. It is hard to find out if this is used by any other plugin. And if some other plugin happens to rely upon it they will break since the class is called something else in the version of the plugin they compile against.

Though in the context of this plugin it is highly unlikely that some other plugin would have such a dependency.

Reply:
I kept it for now in case of complains we can change it back. The reason why I changed the name is that it conflicts with the name of the new `notification` files.